chessington/engine/pieces.py

```diff
@@ -33,14 +33,45 @@ class Pawn(Piece):
     """
     A class representing a chess pawn.
     """
+
+    def get_moves_by_player_color(self):
+        move_forward_once = 1
+        move_forward_twice = 2
+        row_first_time_move_forward_twice = 1
+        if self.player == Player.BLACK:
+            move_forward_once = -1
+            move_forward_twice = -2
```

Comment:
For `move_forward_once` and `move_forward_twice` you can use a [ternary operator](https://book.pythontips.com/en/latest/ternary_operators.html) and multiply by `-1` to determine the value.